frontend/src/static/js/v_zoom.js

```diff
@@ -133,11 +135,6 @@ window.vZoom = {
       }
     },
 
-    restoreZoomTab() {
-      // restore selected user's commits and file type colors from v_summary
-      this.$root.$emit('restoreCommits', this.info);
```

Comment:
Since this method is not used, should it also be removed?

Reply:
I don't understand what you mean. The method is not used so it has already been removed?

Reply:
My bad, didn't see this.
![image](https://user-images.githubusercontent.com/35621726/109971963-bea0a280-7d31-11eb-946a-9a06d37c5e8a.png)